tests/Feature/PhotosMultiBytesTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * We don't care for unhandled exceptions in tests.
+ * It is the nature of a test to throw an exception.
+ * Without this suppression we had 100+ Linter warning in this file which
+ * don't help anything.
+ *
+ * @noinspection PhpDocMissingThrowsInspection
+ * @noinspection PhpUnhandledExceptionInspection
+ */
+
+namespace Tests\Feature;
+
+use Tests\Feature\Base\PhotoTestBase;
+use Tests\TestCase;
+
+class PhotosMultiBytesTest extends PhotoTestBase
```

Comment:
Why has the test been put into its own class? I doubt that this class with this special name will be extended by any other test in the future. So why not adding the test to one of the existing classes.

Reply:
Fixed it myself in https://github.com/LycheeOrg/Lychee/commit/650c2577394c6bdf980c842b472af9e8b08e63e0.